db/seeds.rb

```diff
@@ -1,7 +1,124 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Piece.delete_all
+Review.delete_all
+User.delete_all
```

Comment:
This is destructive. What happens if you run your seed file after you have dedicated users of your website? If you haven't already, please check out the reading on [Seeding your database](https://learn.launchacademy.com/lessons/seeding).